### DISCUSSION TITLE
https://fnoobt.github.io/posts/text-and-typography/

### DISCUSSION THREAD

**Discussion by giscus** [fnoobt/giscus#1] [Announcements]
# https://fnoobt.github.io/posts/text-and-typography/

这篇文章是为了展示帖子的Markdown语法渲染，你也可以用它作为写作的例子。现在，让我们开始查看文本和排版。

https://fnoobt.github.io/posts/text-and-typography/

**Comment by fnoobt** [comment]
giscus评论测试

**Reply by fnoobt** [reply]
回复测试
